src/guides/v2.3/extension-dev-guide/extension_attributes/adding-attributes.md

```diff
@@ -106,14 +106,20 @@ Likewise, the `afterSave` plugin should manipulate the entity data before return
 ```php
 public function afterSave
 (
-    \Magento\Catalog\Api\ProductRepositoryInterface $subject,
-    \Magento\Catalog\Api\Data\ProductInterface $entity
+     \Magento\Catalog\Api\ProductRepositoryInterface $subject,
+     \Magento\Catalog\Api\Data\ProductInterface $result, /** result from the save call **/
```

Comment:
Normally in PHP single line comments are done with // e.g.` // result from the save call`
Is there a reason the /** .... **/ format is used?

Reply:
no, I will update it thanks.

Reply:
No - if it's being used elsewhere then you should stick to it for convention. It's not a must.